server/common/rest.py

```diff
@@ -25,6 +30,46 @@
 from server.common.genesets import summarizeQueryHash
 from server.common.fbs.matrix import decode_matrix_fbs
 
+def get_image_spatial(request, data_adaptor):
```

Comment:
I think it should be called `image_spatial_get` or `spatial_image_get` if following the function name schema in this file.